### ISSUE TITLE
Issue with table with accents in 1.10.0

### ISSUE THREAD

**Issue by alextekartik** [simolus3/sqlite3.dart#148]
Hello, you know I'm an active use of your libs ! I had a CI failure in sqflite_common_ffi after upgrading to 1.10.0. I was able to summarize it into the following (stupid) test:

```dart
test('accent', () async {
  var db = sqlite3.open(':memory:');
  var table = 'télé'; // with accent
  var column = 'mycolumn';
  db.execute('''
    CREATE TABLE $table (
      $column INTEGER
    )''');
  db.execute('INSERT INTO $table($column) VALUES(1)');
  var statement = db.prepare('SELECT * FROM $table');
  var cursor = statement.selectCursor();
  cursor.moveNext();
  print(cursor.current);
  print(cursor.current.values);

  db.dispose();
});
```

This works on previous version but fails with sqflite3 1.10.0 

While the following works:
```dart
db.execute('INSERT INTO $table($column) VALUES(1)');
```
It fails at:
```dart
db.prepare('SELECT * FROM $table')
```

with the following errors:
```
dart:core                                                _StringBase.substring
package:sqlite3/src/implementation/database.dart 311:36  DatabaseImplementation._prepareInternal
package:sqlite3/src/implementation/database.dart 358:19  DatabaseImplementation.prepare
package:sqlite3/src/ffi/implementation.dart 111:18       FfiDatabaseImplementation.prepare
test/raw_sqlite3_test.dart 14:24                         main.<fn>

RangeError (end): Invalid value: Not in inclusive range 0..18: 20
```

This is not a critical issue (who's gonna put accent in a table name besides french) but I thought it was better informing you first to see if there is any workaround or encoding issue to fix. Thanks !

**Comment by alextekartik** [comment]
FYI I have a test that tries the following table and column name: '[](){}:;?/\\&éçà^ù*-+,!̣' and it is working in pre 1.10.0

**Comment by simolus3** [comment]
Thanks a lot for the report, there's no need to belittle this issue :) The package should definitely support characters that get mapped to multiple bytes in utf8, so this is a very serious regression. I have just published version 1.10.1, which fixes the problem.
